src/pbview.cpp

```diff
@@ -193,12 +192,9 @@ void PbView::run(bool auto_download, bool wrap_scroll)
 		case OP_PB_DOWNLOAD: {
 			if (ctrl->downloads().size() >= 1) {
 				const auto idx = downloads_list.get_position();
-				if (ctrl->downloads()[idx].status() !=
-					DlStatus::DOWNLOADING) {
-					std::thread t{PodDlThread(
-							&ctrl->downloads()[idx],
-							ctrl->get_cfgcont())};
-					t.detach();
+				auto item = ctrl->downloads()[idx];
```

Comment:
`auto` drops references, so `item` here is a `Download`, and we're passing a reference to this local variable to the thread in `start_download()`. I think this should be changed to `auto&`.

Reply:
Thanks for catching that!
Fixed by amending/rebasing: https://github.com/newsboat/newsboat/pull/1382/commits/4acab0faf99451c73b4101c631ca353c5f79b058#diff-8ba3707b9e587217ca7dfc65695eae819d8c1bd28e10ff720484cd6033dab44aR195